### PR TITLE
git-*: use inclusive language

### PR DESCRIPTION
**Pull request by sbrl**
Suggestions for improving on #4459, which I reverted in #4532.

/cc everyone involved: @owenvoke, @mebeim, @navarroaxel, @qvissak, @agnivade.

### CODE REVIEW

File: pages.es/common/git-rebase.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 > Se utiliza comúnmente para "mover" una rama entera a otra base, ya que crea copias de los commits en una nueva ubicación.
 > Más información: <https://git-scm.com/docs/git-rebase>.
 
-- Rebasa la rama actual en lo más alto de la rama master:
+- Rebasa la rama actual en lo más alto de la rama branch_name:
```
Comment (mebeim) [inline pages.es/common/git-rebase.md:7]:
```suggestion
- Rebasa la rama actual en lo más alto de una rama:
```
Change "the branch branch_name" to "a branch".

**Review by mebeim — CHANGES_REQUESTED** [review]
Minor nitpick for the one spanish page, otherwise looks good to me.

File: pages.es/common/git-rebase.md (hunk 1 above)
Comment (sbrl) [reply pages.es/common/git-rebase.md:7]: Thanks! I forgot to mention that the `branch_name` needed translating here, since I don't speak the language haha

File: pages.es/common/git-rebase.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
 > Se utiliza comúnmente para "mover" una rama entera a otra base, ya que crea copias de los commits en una nueva ubicación.
 > Más información: <https://git-scm.com/docs/git-rebase>.
 
-- Rebasa la rama actual en lo más alto de la rama master:
+- Rebasa la rama actual en lo más alto de una rama:
```
Comment (navarroaxel) [inline pages.es/common/git-rebase.md:7]: I'm from Argentina but I like to read docs in English :grimacing:.  `Rebasa` is the worst Spanish translation for `rebase` lol. The [official git](https://git-scm.com/book/es/v2/Ramificaciones-en-Git-Reorganizar-el-Trabajo-Realizado) use `reorganización`.
```suggestion
- Reorganiza la rama actual en lo más alto de otra rama:
```

**Review by qvissak — APPROVED** [review]

File: pages.es/common/git-rebase.md [hunk 3]
```diff
@@ -4,9 +4,9 @@
...
 
-`git rebase {{master}}`
+`git rebase {{branch_name}}`
```
Comment (waldyrious) [inline pages.es/common/git-rebase.md:9]: It's unclear whether `branch_name` refers to the current branch or the target branch. We might want to use a more explicit token name here.

File: pages.es/common/git-rebase.md [hunk 4]
```diff
@@ -34,4 +34,4 @@
 
 - Resuelve automáticamente cualquier conflicto favoreciendo la versión de la rama en la que se esta trabajando (en este caso la palabra `theirs` tiene un significado invertido):
 
-`git rebase -X theirs {{master}}`
+`git rebase -X theirs {{branch_name}}`
```
Comment (waldyrious) [inline pages.es/common/git-rebase.md:37]: I'm wondering if we shouldn't translate "branch_name" to Spanish, as there's no need for it to be in English.

File: pages/common/git-rebase.md [hunk 5]
```diff
@@ -4,9 +4,9 @@
 > Commonly used to "move" an entire branch to another base, creating copies of the commits in the new location.
 > More information: <https://git-scm.com/docs/git-rebase>.
 
-- Rebase the current branch on top of the master branch:
+- Rebase the current branch on top of the another specified branch:
 
-`git rebase {{master}}`
+`git rebase {{branch_name}}`
```
Comment (waldyrious) [inline pages/common/git-rebase.md:9]: As mentioned above, 

> It's unclear whether `branch_name` refers to the current branch or the target branch. We might want to use a more explicit token name here.

File: pages/common/git-subtree.md [hunk 6]
```diff
@@ -5,19 +5,19 @@
...
-`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{master}}`
+`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{branch_name}}`
 
 - Update subtree repository to its latest commit:
 
-`git subtree pull --prefix={{path/to/directory/}} {{repository_url}} {{master}}`
+`git subtree pull --prefix={{path/to/directory/}} {{repository_url}} {{branch_name}}`
 
-- Merge a subtree repository into master:
+- Merge a subtree repository into a specified branch:
```
Comment (waldyrious) [inline pages/common/git-subtree.md:14]: I'm not sure if the last parameter refers to the source or the target branch. As the [author of this page](https://github.com/tldr-pages/tldr/pull/4012), could you comment, @einverne?

File: pages/common/git-subtree.md [hunk 7]
```diff
@@ -5,19 +5,19 @@
 
 - Add a git repository as a subtree:
 
-`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{master}}`
+`git subtree add --prefix={{path/to/directory/}} --squash {{repository_url}} {{branch_name}}`
```
Comment (waldyrious) [inline pages/common/git-subtree.md:8]: Is the final parameter mandatory? Maybe we should omit it in most examples if it defaults to the primary branch, which I'd suppose is what most people would want.

File: pages.es/common/git-rebase.md (hunk 4 above)
Comment (mebeim) [reply pages.es/common/git-rebase.md:37]: You're definitely right, the token should be translated.

File: pages/common/git-rebase.md (hunk 5 above)
Comment (navarroaxel) [reply pages/common/git-rebase.md:9]:
```suggestion
`git rebase {{base_branch}}`
```

File: pages.es/common/git-rebase.md (hunk 3 above)
Comment (navarroaxel) [reply pages.es/common/git-rebase.md:9]:
```suggestion
`git rebase {{rama_base}}`
```

File: pages.es/common/git-rebase.md (hunk 4 above)
Comment (navarroaxel) [reply pages.es/common/git-rebase.md:37]:
```suggestion
`git rebase -X theirs {{base_branch}}`
```

File: pages.es/common/git-rebase.md (hunk 3 above)
Comment (waldyrious) [reply pages.es/common/git-rebase.md:9]: In English, I'd go with `target_branch` or `new_base_branch`. That's because the very notion of rebasing implies two bases, the old one and the new one, so merely base_branch would still be ambiguous as it could refer to either base. How would you suggest we express this concept in Spanish?

File: pages.es/common/git-rebase.md (hunk 4 above)
Comment (waldyrious) [reply pages.es/common/git-rebase.md:37]: I don't understand your suggestion, @navarroaxel — can you clarify the reasoning for it?

File: pages/common/git-rebase.md (hunk 5 above)
Comment (waldyrious) [reply pages/common/git-rebase.md:9]: I'll have to quote myself again (the two discussions are the same, only in two different languages):

> I'd go with `target_branch` or `new_base_branch`. That's because the very notion of rebasing implies two bases, the old one and the new one, so merely `base_branch` would still be ambiguous as it could refer to either base.

File: pages.es/common/git-rebase.md (hunk 3 above)
Comment (navarroaxel) [reply pages.es/common/git-rebase.md:9]: Re-reading again [the docs](https://git-scm.com/book/es/v2/Ramificaciones-en-Git-Reorganizar-el-Trabajo-Realizado) they use the term:
```suggestion
`git rebase {{rama_de_reorganización}}`
```

File: pages/common/git-rebase.md (hunk 5 above)
Comment (navarroaxel) [reply pages/common/git-rebase.md:9]: Reading [the docs](https://git-scm.com/book/en/v2/Git-Branching-Rebasing#_more_interesting_rebases:~:text=More%20Interesting%20Rebases), in `More Interesting Rebases` section, they use the term:
```suggestion
`git rebase {{rebase_target_branch}}`
```

File: pages/common/git-subtree.md (hunk 7 above)
Comment (qvissak) [reply pages/common/git-subtree.md:8]: I would say that is outside the scope of this PR.

**Comment by qvissak** [comment]
To combat ambiguity, this is why in my original PR I used `main` since this is the new official terminology by [GitHub](https://github.com/github/renaming), and the direction that [GitLab](https://gitlab.com/gitlab-org/gitlab/-/issues/222204) and [BitBucket](https://jira.atlassian.com/browse/BCLOUD-20212) will likely take. BitBucket, however, is waiting for `git init` to change the default branch from `master` to `main`.

I think by definition, the word `main` is clear. No need to guess what `base_branch`, `target_branch`, `rebase_target_branch`, etc. means.

File: pages.es/common/git-rebase.md (hunk 4 above)
Comment (sbrl) [reply pages.es/common/git-rebase.md:37]: Yeah, we should definitely translate here. I just inserted it as a suggestion as I unfortunately don't know the language

File: pages/common/git-rebase.md (hunk 5 above)
Comment (sbrl) [reply pages/common/git-rebase.md:9]: Great point, @waldyrious. `new_base_branch` sounds the most clear to me, so how about we go with that?

File: pages/common/git-subtree.md (hunk 7 above)
Comment (sbrl) [reply pages/common/git-subtree.md:8]: It doesn't hurt to review it now I don't think, @qvissak.

Anyway, it looks like only the repository URL is optional according to the man page @waldyrious, as I see the following in the usage summary at the top:

```
git subtree add   -P <prefix> <commit>
git subtree add   -P <prefix> <repository> <ref>
```

You'd have thought it would be the commit ref that's optional, but it's the repository url instead :woman_shrugging:

Interestingly we mention here `branch_name`, but it's actually more like a `commit_ish` according to the man page. That said, I havevn't actually used this command personally, so I wouldn't know.

File: pages/common/git-subtree.md (hunk 6 above)
Comment (sbrl) [reply pages/common/git-subtree.md:14]: Looking at the man page, it details this:

> Merge recent changes up to <commit> into the <prefix> subtree.

So I think that it's actually going in the opposite direction to that of the description here.

Again, I haven't actually used the command before, so I'm not sure. Let's wait for @einverne's response.

Comment (einverne) [reply pages/common/git-subtree.md:14]: According to the document, it says:

> git subtree merge -P <prefix> <commit>
> Merge recent changes up to <commit> into the <prefix> subtree.

What the `subtree merge` do is to merge the latest commit to the subtree.

Comment (sbrl) [reply pages/common/git-subtree.md:14]: Ohhh I see. Thanks for the insight here, @einverne! So could we reword to `{{subtree_branch_name}}` here then perhaps?

**Comment by sbrl** [comment]
Oh I see, @qvissak! I haven't actually seen that being the new official terminology yet - as I don't create new repositories very often - hence the confusion :P

Either way, I think replacing the concrete example here of a branch name with something more descriptive will aid clarity too.

File: pages/common/git-subtree.md (hunk 6 above)
Comment (einverne) [reply pages/common/git-subtree.md:14]: @sbrl Yes, and the origin description may not be accurate. How about

> Merge recent changes up to {{latest_subtree_commit}} into the subtree.

File: pages/common/git-rebase.md (hunk 5 above)
Comment (sbrl) [reply pages/common/git-rebase.md]: Applied `new_base_branch`, so marking this as resolved. Feel free to unresolve if this isn't the case.

File: pages/common/git-subtree.md (hunk 6 above)
Comment (sbrl) [reply pages/common/git-subtree.md]: Applied suggestion, @einverne.

Is this better?

File: pages/common/git-subtree.md (hunk 7 above)
Comment (sbrl) [reply pages/common/git-subtree.md:8]: Marking as resolved, but feel free to unresolve if this isn't the case.

File: pages.es/common/git-rebase.md (hunk 4 above)
Comment (waldyrious) [reply pages.es/common/git-rebase.md:37]: @navarroaxel would `{{nombre_de_rama}}` or simply `{{rama}}` work here?

File: pages.es/common/git-rev-list.md [hunk 8]
```diff
@@ -9,7 +9,7 @@
 
 - Muestra los commits más reciente a partir de una fecha y una rama específica:
 
-`git rev-list --since={{'2019-12-01 00:00:00'}} {{master}}`
+`git rev-list --since={{'2019-12-01 00:00:00'}} {{branch_name}}`
```
Comment (waldyrious) [inline pages.es/common/git-rev-list.md:12]: Same as [above](https://github.com/tldr-pages/tldr/pull/4533/files#r500594375) — this should be translated.

File: pages.es/common/git-revert.md [hunk 9]
```diff
@@ -13,7 +13,7 @@
 
 - Revierte múltiples commits:
 
-`git revert {{master~5..master~2}}`
+`git revert {{branch_name~5..branch_name~2}}`
```
Comment (waldyrious) [inline pages.es/common/git-revert.md:16]: Same as [above](https://github.com/tldr-pages/tldr/pull/4533/files#r500594375) — these should be translated.

File: pages.es/common/git-rebase.md (hunk 4 above)
Comment (navarroaxel) [reply pages.es/common/git-rebase.md:37]: I think rama is OK:
```suggestion
`git rebase -X theirs {{rama}}`
```
but if we use `rama_de_reorganización` above, this is another option:
```suggestion
`git rebase -X theirs {{rama_de_reorganización}}`
```
If you now English `theirs {{rama}}` has a lot of sense, otherwise `theirs {{rama_de_reorganización}}` is better.

File: pages.es/common/git-rev-list.md (hunk 8 above)
Comment (navarroaxel) [reply pages.es/common/git-rev-list.md:12]:
```suggestion
`git rev-list --since={{'2019-12-01 00:00:00'}} {{nombre_de_rama}}`
```

File: pages.es/common/git-revert.md (hunk 9 above)
Comment (navarroaxel) [reply pages.es/common/git-revert.md:16]:
```suggestion
`git revert {{rama~5..rama~2}}`
```

**Review by mebeim — APPROVED** [review]
I've applied @navarroaxel suggestions for pages under `pages.es`. LGTM now!

File: pages/common/git-rebase.md [hunk 10]
```diff
@@ -4,9 +4,9 @@
 > Commonly used to "move" an entire branch to another base, creating copies of the commits in the new location.
 > More information: <https://git-scm.com/docs/git-rebase>.
 
-- Rebase the current branch on top of the master branch:
+- Rebase the current branch on top of the another specified branch:
```
Comment (waldyrious) [inline pages/common/git-rebase.md:7]:
```suggestion
- Rebase the current branch on top of another specified branch:
```

File: pages.es/common/git-rev-list.md [hunk 11]
```diff
@@ -9,7 +9,7 @@
 
 - Muestra los commits más reciente a partir de una fecha y una rama específica:
```
Comment (waldyrious) [inline pages.es/common/git-rev-list.md:10]: This is not strictly related to this PR, but while we're at it, we should fix the typo:
```suggestion
- Muestra los commits más recientes a partir de una fecha y una rama específica:
```

@navarroaxel can you confirm please?

**Review by waldyrious — CHANGES_REQUESTED** [review]
Just a couple of additional tweaks and should be good to go!

File: pages.es/common/git-rev-list.md (hunk 11 above)
Comment (navarroaxel) [reply pages.es/common/git-rev-list.md:10]: Yep, it's good.

**Comment by sbrl** [comment]
Suggestions applied, @waldyrious :-)

**Review by waldyrious — APPROVED** [review]
Alright, let's go! 🚀🚀
